pom.xml

```diff
@@ -118,9 +118,9 @@
             <version>${jackson.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.codehaus.woodstox</groupId>
-            <artifactId>woodstox-core-asl</artifactId>
-            <version>4.1.6</version>
+            <groupId>com.fasterxml.woodstox</groupId>
+            <artifactId>woodstox-core</artifactId>
+            <version>6.5.1</version>
```

Comment:
why this version? there are several newer, up to 7.1.0?

Reply:
I think I pulled the version out of the library it was conflicting with. I didn't want to create a different dependency issue. Are you ok with that?